.github/workflows/rust.yml

```diff
@@ -84,12 +84,21 @@ jobs:
         go install github.com/bazelbuild/buildtools/buildifier@v0.0.0-20241212155839-a9c248f4b684
         export PATH=$PATH:$(go env GOPATH)/bin
 
+    - name: Check MODULE.bazel.lock
+      run: bazelisk mod deps --lockfile_mode=error
+
```

Comment:
Could you move this to the end of the job and rename it to `Format (MODULE.bazel.lock)` to match existing jobs?